JobConfig/reco/prolog.fcl

```diff
@@ -130,14 +130,15 @@ Reconstruction : {
       caloShowerSimTag : ""
       caloShowerROTag : ""
       caloClusterMCTag : "CaloClusterTruthMatch"
-      caloHitMCTag : "CaloHitTruthMatch"
+#      caloHitMCTag : "CaloHitTruthMatch"
```

Comment:
I'm not sure why CaloHitMCTag is commented out.  Is it because the step filter already does this?  I think these need to be re-written anyways, just because the pointers change with the CaloClusterMCs are compressed.

Reply:
The new validated fhicl was throwing an error because this fhicl parameter isn't defined. I thought about adding it but it looked like we don't use caloHitMCTag anywhere. I think we just get to CaloHitMCs through CaloClusterMCs but I meant to look into this further in case it was a rebase error on my part.